0028-web-monetization/0028-web-monetization.md

```diff
@@ -44,24 +44,30 @@ This flow refers to the user's **browser** and the user's **provider**, [defined
 - The user navigates their browser to a webpage.
 - The browser sets `document.monetization` to an Object which implements [EventTarget](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget).
 - The browser sets `document.monetization.state` to `pending`.
-- The browser looks for the Web Monetization `<meta>` tags ([specified below](#meta-tags)). The `<meta>` tags MUST NOT be inserted dynamically using client-side Javascript.
+- The browser observes the `<head>` and looks for a Web Monetization `<meta name="monetization">` tag ([specified below](#meta-tags)).
+  - There MUST be only one `<meta name="monetization">` tag at any given time
   - The `<meta>` tags MUST be in the `<head>` of the document.
-  - If the Web Monetization `<meta>` tags are malformed, the browser will stop here. The browser SHOULD report a warning via the console.
-  - If the Web Monetization `<meta>` tags are well-formed, the browser should extract the Payment Pointer.
-  - The browser will generate a fresh UUID (version 4) and use this as the Request ID from this point forward. **This Request ID MUST be unique per page load**, not per browser, session nor site.
+  - The `<meta>` tags MUST be in the top level window (i.e. not inside an iframe)  
+  - If any of the Web Monetization `<meta>` tags are malformed, the browser will stop here. The browser SHOULD report a warning via the console.
+  - If the Web Monetization `<meta name="monetization">` tag is well-formed, the browser should extract the Payment Pointer or SPSP url
+  (TODO: This would imply that when dynamically injecting multiple meta tags (in the sense of others than `<meta name="monetization">`, it would require careful ordering so as to be operating on the information defined by the set of tags as a whole, removal with document.head.querySelectorAll('meta[name^="monetization"]'), then adding the name="monetization" tag last)
```

Comment:
Should the document portions here be code blocks?

Reply:
That's just some scribble thoughts I put in for discussion/thought ... 

People will have to be careful to make sure they have all their ducks (monetization meta tags used to convey info other than a payment pointer) in a row before updating.

Reply:
oh its all in a todo